Modules/SubModule_Import/xASL_imp_Initialize.m

```diff
@@ -94,26 +126,30 @@
         imPar.bMatchDirectories  = false;
     end
 
-    %% 5. Specify the additional details of the conversion
-    if ~isfield(imPar,'bVerbose') || isempty(imPar.bVerbose)
-        imPar.bVerbose = true;
-    end
-    if ~isfield(imPar,'bOverwrite') || isempty(imPar.bOverwrite)
-        imPar.bOverwrite  = false; % NB, the summary file will be recreated anyway and dicom conversion in temp is always done, even if dest. exists
-    end
-    if ~isfield(imPar,'visitNames') || isempty(imPar.visitNames)
-        imPar.visitNames = {};
-    end
-    if ~isfield(imPar,'nMaxVisits') || isempty(imPar.nMaxVisits)
-        imPar.nMaxVisits = 0;
-    end
-    if ~isfield(imPar,'sessionNames') || isempty(imPar.sessionNames)
-        imPar.sessionNames = {};
-    end
-    if ~isfield(imPar,'nMaxSessions') || isempty(imPar.nMaxSessions)
-        imPar.nMaxSessions = 0;
+end
+
+
+%% Warn the user if token aliases are invalid according to our pipeline/BIDS
+function xASL_imp_InitializeCheckTokens(imPar)
+
+    % Check tokenScanAliases
+    if isfield(imPar,'tokenScanAliases')
+        if size(imPar.tokenScanAliases,2)==2
+            for iToken = 1:size(imPar.tokenScanAliases,1)
+                currentToken = imPar.tokenScanAliases{iToken,2};
+                % Warn if the token contains ASL but is not ASL4D
+                if ~isempty(regexp(currentToken,'ASL', 'once')) && ~strcmp(currentToken,'ASL4D')
```

Comment:
`regexp` should be `regexpi` here
`'once'` can be omitted
(same for T1w below)

Reply:
Once is recommended by matlab, agree on the regexpi

Reply:
Agree - fixed.

Reply:
Fixed...